### PR TITLE
[doc] Improve the dev install doc

### PR DESCRIPTION
**Pull request by k-ye**
Related issue = #

### CODE REVIEW

**Review by Vissidarte-Herman — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # Contributing Guide
 
-See the [Contribution Guidelines](https://docs.taichi.graphics/lang/articles/contribution/contributor_guide) on our documentation site.
+Thank you for your interest in contributing to Taichi! Please check out the [Contribution Guidelines](https://docs.taichi.graphics/lang/articles/contribution/contributor_guide) for how to make a contribution.
+
+## Developer Installation
+
+Please follow the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) doc to learn how to build Taichi from scratch.
```
[the remark this answers is not in the document]
Comment (k-ye) [reply CONTRIBUTING.md:7]:
```suggestion
Please follow the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) to build Taichi from scratch.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,4 +4,4 @@ Thank you for your interest in contributing to Taichi! Please check out the [Con
 
 ## Developer Installation
 
-Please follow the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) doc to learn how to build Taichi from scratch.
+Taichi is mainly developed in C++17 and Python3. Please check out the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) to build Taichi from scratch. Note that Taichi currently depends on LLVM-10.0.0, and we recommned to use [our prebuilt LLVM libraries](https://docs.taichi.graphics/lang/articles/contribution/dev_install#installing-dependencies) for each platform.
```
Comment (Vissidarte-Herman) [inline CONTRIBUTING.md:7]: Taichi is developed mainly in C++17 and Python3. Please check out the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) to build Taichi from scratch. Note that Taichi is LLVM-10.0.0 dependent and that we recommend installing [our pre-built LLVM libraries](https://docs.taichi.graphics/lang/articles/contribution/dev_install#installing-dependencies) for your platform.

File: README.md [hunk 3]
```diff
@@ -47,9 +47,9 @@ Please build from source for other configurations (e.g., your CPU is ARM, or you
  - On Windows, please install [Microsoft Visual C++ Redistributable](https://aka.ms/vs/16/release/vc_redist.x64.exe) if you haven't.
  - [[All releases]](https://github.com/taichi-dev/taichi/releases)
 
-## Developer Installation
+## Contributing
 
-Please follow [this doc](https://docs.taichi.graphics/lang/articles/contribution/dev_install) to learn how to build Taichi from source. Note that Taichi requires LLVM-10.0.0, and it is recommneded to use [our prebuilt LLVM libraries](https://docs.taichi.graphics/lang/articles/contribution/dev_install#installing-dependencies) for each platform.
+We welcome feedback and comments! If you would like to contribute to Taichi, please check out [CONTRIBUTING.md](CONTRIBUTING.md).
```
Comment (Vissidarte-Herman) [inline README.md:52]: We'd love to hear your comments or any of your feedback! If you would like to contribute to Taichi, please check out the [Contribution Guidelines](CONTRIBUTING.md) first.

Comment (Vissidarte-Herman) [reply README.md:52]: Off-topic: Just so you know, contributing.md only holds a link to another contribution guideline on the doc site.

**Review by Vissidarte-Herman — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (Vissidarte-Herman) [reply CONTRIBUTING.md:7]: Shall we be more explicit by saying "build Taichi from source"? IMO, this is more commonplace?

Comment (k-ye) [reply CONTRIBUTING.md:7]:
```suggestion
Taichi is developed mainly in C++17 and Python3. Please check out the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) to build Taichi from scratch. Note that Taichi is LLVM-10.0.0 dependent and that we recommend installing [our pre-built LLVM libraries](https://docs.taichi.graphics/lang/articles/contribution/dev_install#installing-dependencies) for your platform. 
```

File: README.md (hunk 3 above)
Comment (k-ye) [reply README.md:52]:
```suggestion
We'd love to hear your comments or any of your feedback! If you would like to contribute to Taichi, please check out the [Contribution Guidelines](CONTRIBUTING.md) first. 
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
 # Contributing Guide
 
-See the [Contribution Guidelines](https://docs.taichi.graphics/lang/articles/contribution/contributor_guide) on our documentation site.
+Thank you for your interest in contributing to Taichi! Please check out the [Contribution Guidelines](https://docs.taichi.graphics/lang/articles/contribution/contributor_guide) for how to make a contribution.
+
+## Developer Installation
+
+Taichi is developed mainly in C++17 and Python3. Please check out the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) to build Taichi from scratch. Note that Taichi is LLVM-10.0.0 dependent and that we recommend installing [our pre-built LLVM libraries](https://docs.taichi.graphics/lang/articles/contribution/dev_install#installing-dependencies) for your platform.
```
Comment (k-ye) [inline CONTRIBUTING.md:7]:
```suggestion
Taichi is developed mainly in C++17 and Python3. Please check out the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) to build Taichi from source. Note that Taichi is LLVM-10.0.0 dependent and that we recommend installing [our pre-built LLVM libraries](https://docs.taichi.graphics/lang/articles/contribution/dev_install#installing-dependencies) for your platform.
```

**Review by Vissidarte-Herman — APPROVED** [review]

**Pull request merged**